### ISSUE TITLE
1.1 does not have the basedir config default value

### ISSUE THREAD

**Issue by ndintenfass** [BKWLD/laravel-pug#5]
For reasons I can't explain my /config/pug.php is not getting picked up at render time. When I `tinker` in the terminal the configurations seem to be there, but when I load the page I get the "The 'basedir' option need to be set to use absolute path like..." message. My fix was to manually add the line that's in the current `master` branch in config.php that has the basedir, but that does not appear in the 1.1 release. A quick solution would be to do a release that has the basedir in the config, though I'm still not clear why my config isn't loading (I'm new to Laravel, so learning how to debug such things in order to solve this problem). I'm on Laravel 5.3.29.

**Comment by Riant** [comment]
Same problem to me.

I fixed that with a config file in laravel config folder called "laravel-pug.php", So you can add a file "/config/laravel-pug.php", set content as below, then set pug options to be here.

``` php
<?php return [

    /**
     * Passthrough php-pug config options
     * https://github.com/pug-php/pug/blob/master/src/Jade/Jade.php
     */

    // Required for extending layouts
    'basedir' => resource_path('views'),

    // 'cache'              => null,
    // 'stream'             => null,
    // 'extension'          => array('.pug', '.jade'),
    // 'prettyprint'        => false,
    // 'phpSingleLine'      => false,
    // 'keepBaseName'       => false,
    // 'allowMixinOverride' => true,
    // 'allowMixedIndent'   => true,
    // 'keepNullAttributes' => false,
    // 'restrictedScope'    => false,
    // 'singleQuote'        => false,
    // 'filterAutoLoad'     => true,
    // 'indentSize'         => 2,
    // 'indentChar'         => ' ',
    // 'customKeywords'     => array(),
    // 'classAttribute'     => null,
];
```

**Comment by weotch** [comment]
@thelucre Lets look at this when you get in and make sure the config file is named what we want it to be.

**Comment by weotch** [comment]
Fixed in #8
